### DISCUSSION TITLE
Nonlinear analysis

### DISCUSSION THREAD

**Discussion by epsi1on** [BriefFiniteElementNet/BriefFiniteElement.Net#104] [Ideas]
Continue from Issue #6 (Nonlinear analysis) and #36

**Comment by xyont** [comment]
hi,

Nonlinear analysis can have a large branch of topics, however there's several entrance level known as :

- P (big & small) delta effect of Frame members, related to geometric stiffness matrix and Timoshenko beam for accuracy improvement.
- Tension-only members, i.e steel brace subjected to reversed lateral loads.
- Spring elements with compression-only, i.e beam or plate on elastic foundation due to large overturning moment.

hopefully, all above useful features are interested to implementing in the future of development.

regards,

**Reply by epsi1on** [reply]
Good, thanks for information.
Actually nonlinear is doing as separated project: https://github.com/BriefFiniteElementNet/NLBriefFiniteElement.Net/
we could continue there

**Reply by Aliturkisk** [reply]
Good news, any update on nonlinear. Are there a sample?

**Reply by epsi1on** [reply]
Hello,
It is still in design phase and real library is not present, after design phase, the code will be implemented and after implementation of real library, the samples will be available. :)
The work is very slow on the library. so unfortunately not sure when it will be available...
